app/views/host_initiator/show_policy_timeline.html.haml

```diff
@@ -0,0 +1 @@
+#main_div= render :partial => "layouts/tl_show"
```

Comment:
@llivne why do we need to render this view, i dont see any toolbars related to this new screen

Reply:
@h-kataria we dont, I removed it (I copy/paste a template and I guess I forgot to remove it)